### PR TITLE
[Reviewer: Adam] Fix up FVs

### PR DESCRIPTION
**Pull request by eleanor-merry**
Fix to store the pids of the Chronos processes rather than use the result of Popen. We need this because https://github.com/Metaswitch/chronos/pull/246 changed how Chronos daemonizes

### CODE REVIEW

File: scripts/chronos_resync.py [hunk 1]
```diff
@@ -115,30 +116,40 @@ def run_app():
...
 
-    sleep(2)
+        f = open(PID_PATTERN % i)
+        processes.append(f.read())
+        f.close()
 
 
 def kill_nodes(lower, upper):
-    # kill nodes with indexes [lower, upper)
+    # Kill nodes with indexes [lower, upper)
```
Comment (johadalin) [inline scripts/chronos_resync.py]: Mismatched brackets

File: scripts/chronos_resync.py [hunk 2]
```diff
@@ -55,6 +55,7 @@
 # perform scaling operations, and check that the correct number of
 # timers still pop
 CHRONOS_BINARY = 'build/bin/chronos'
+PID_PATTERN = 'scripts/pid/chronos.livetest.%i.pid'
```
Comment (johadalin) [inline scripts/chronos_resync.py]: If this is all to do with FV tests, how much work is it to change these references to livetest, to match the class rename below?

Comment (eleanor-merry) [inline scripts/chronos_resync.py]: Will do

**Comment by johadalin** [comment]
Discussed comments, and now happy i follow it all. 
Otherwise, all looks good

File: scripts/chronos_resync.py (hunk 1 above)
Comment (eleanor-merry) [inline scripts/chronos_resync.py]: http://stackoverflow.com/questions/4396290/what-does-this-square-bracket-and-parenthesis-bracket-notation-mean-first1-last

**Pull request merged**